### DISCUSSION TITLE
Our Solar System

### DISCUSSION THREAD

**Discussion by Meghansaha** [qmd-lab/closeread#89] [Show and tell]
![image](https://github.com/user-attachments/assets/dc9b14c5-a502-4ff3-a765-348528968b7a)

Just wanted to share my hacky test drive that's on my GitHub [here](https://meghansaha.github.io/the_solar_system/)!!

@andrewpbray, I had so much fun messing around with this! Please let me know if I can help with testing, docs, anything!!

**Comment by andrewpbray** [comment]
Wow, this is terrific! I love the concept and the way you play around with the narrative blocks and the stickies to build the narrative almost as a dialog.

Thanks for the offer to help! In about a weeks time we're going to start in on a bit of dev, so we'll reach out to see about testing and docs.

🚀 🌔
